poetry.lock

```diff
@@ -293,6 +293,20 @@ python-versions = "*"
 [package.extras]
```

Comment:
Are the changes to lib versions in poetry.lock due to splitting the tests into groups, or are they unrelated changes?

Reply:
Some are related to adding the new dependency, but it looks like it runs the solution engine whenever you add a dependency. We have a handful of deps that are tied to `main` branches on github, which will be resolved fresh - and there may be some path-version updates from other dependencies.